CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-Version 2.0-rc1
-===============
+Version 2.0
+===========
 
 The library has been tested using Agda 2.6.4.
```

Comment:
This line should also change if we're using a different version in CI

Reply:
Also, there's a mention of the version in `doc/installation-guide.md` which should be updated too

Reply:
Thanks, fixed!